docs/docs/how-tos/nebari-aws.md

```diff
@@ -153,12 +153,11 @@ With the `qhub-config.yaml` configuration file now created, Nebari can be deploy
 qhub deploy -c qhub-config.yaml
 ```
 :::note
-If you have account on any [Cloud provider](https://www.nebari.dev/getting-started/cloud-providers) you can create a domain name of your choice, if you are not selecting CloudFlare for [automatic DNS registration](https://www.nebari.dev/how-tos/domain-registry). Taking Cloud Flare as a choice you can create a domain name with some restrictions obviously, for example: `any-domain-name.qhub.dev`. Please keep in mind that after creating your own domain name you need to run  
+During Qhub deployment DNS record is required. You are advised to follow the instructions in the [Nebari DNS](/how-tos/domain-registry.md) section to set up the DNS record and for more information regarding the domain A/CNAME records and how to automatically generate them. If you already have a Cloudflare account, Please add the following flags: `--dns-provider cloudflare` and `--dns-auto-provison` to the Qhub deployment command above and be sure to export your Cloudflare token as commented below:
```

Comment:
During deployment, Qhub will require you to set a DNS record for the domain defined during [initialize](https://deploy-preview-101--nebari-docs.netlify.app/how-tos/nebari-gcp#nebari-initialize). Follow the instructions on [How to set a DNS record for Qhub](/how-tos/domain-registry.md) for an overview of the required steps.